src/main/java/dev/ebrydeu/spring_boot_library/controllers/MessageController.java

```diff
@@ -0,0 +1,21 @@
+package dev.ebrydeu.spring_boot_library.controllers;
+
+import dev.ebrydeu.spring_boot_library.domain.dto.MessageDto;
+import dev.ebrydeu.spring_boot_library.services.MessageService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/messages")
+@RequiredArgsConstructor
+public class MessageController{
+    private final MessageService messageService;
+
+    @PutMapping("/{id}/private/{messagePrivate}")
```

Comment:
```suggestion
    @PatchMapping("/{id}/private/{messagePrivate}")
```
You don't need full update for one field just use partial